scripts/release/package_audit_lib.rb

```diff
@@ -483,9 +502,18 @@ def check_tool_base(path, base_pkgs)
         if package == 'keycloak-js'
           html = File.read('public/js/auth.js')
           html.gsub!(/#{alt_package}-\d+\.\d+\.\d+\.min\.js/, "#{alt_package}-#{latest}.min.js")
-          File.open(filename, 'w') {|file| file.puts html }
+          File.open('public/js/auth.js', 'w') {|file| file.puts html }
         end
       end
     end
   end
 end
+
+def validate_outfile(outfile, package, latest)
+  data = File.read(outfile)
+  if data.length < 100
+    puts "ERROR: While updating #{package} to #{latest} got the following:\n#{data}\n\nCheck the package and version."
+    FileUtils.rm outfile
+    exit 1
+  end
```

Comment:
I added this so this script doesn't just plow ahead when it gets a 404 error from curl and returns an error string instead of the file.